charts/nexus3/Chart.yaml

```
@@ -3,7 +3,7 @@ name: nexus3
 description: Helm chart for Sonatype Nexus 3 OSS.
 type: application
 version: 4.44.0
```

```suggestion
version: 4.44.1
```
This needs bumping too.